app/sigverify.go

```diff
@@ -0,0 +1,43 @@
+package althea
+
+import (
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+	canto "github.com/Canto-Network/Canto/v5/app"
+)
+
+const (
+	secp256k1VerifyCost uint64 = 21000
+)
+
```

Comment:
Canto and Ethermint don't support SDK secp256k1 keys because they 1. don't register them and 2. don't have a gas consumer which accepts the type. This consumer adds a single case for SDK secp256k1 pubkeys and then calls the Canto implementation directly.